crypto/bindings/lookup.ts

```diff
@@ -21,7 +21,13 @@ type LookupInfo = [
   max_joint_size: number,
   features: LookupFeatures
 ];
-type LookupSelectors<PolyComm> = [_: 0, lookup: MlOption<PolyComm>];
+type LookupSelectors<PolyComm> = [
+  _: 0,
+  lookup: MlOption<PolyComm>,
```

Comment:
xor comes first

Reply:
@dannywillems this has to match the type here, where `xor` comes second: https://github.com/MinaProtocol/mina/blob/7a975d31aee7efef5cadc215ea0386a02bdf8808/src/lib/crypto/kimchi_bindings/stubs/kimchi_types.ml#L200-L205

Reply:
Oh :o ... Good catch. Let's open an issue to fix the wrong order, just in case we make some assumptions in the order somewhere.